### ISSUE TITLE
Graphic EQ Slider should be switched via Tab

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#9217]
### Description

The User should be able to use 'Tab' and 'Shift-Tab' to move through the EQ Sliders within the Graphic EQ effect in Audacity 4.

**Comment by DilsonsPickles** [comment]
I've made a prototype to test out this behaviour. 

It can be found [here](https://graphic-eq-tabbing.netlify.app/).

I'll bring this to a future design team meeting for discussion.

**Comment by DilsonsPickles** [comment]
Updated prototype following feedback

[Link to prototype](https://graphic-eq-tabbing.netlify.app/)

Click on the "Open Graphical EQ" blue button (The other button summons an older version). 

Using arrow keys on the fader sliders now also summons and renders the tooltip.

**Comment by DavidBailes** [comment]
> Updated prototype following feedback

@DilsonsPickles .
For the bottom row of buttons in the dialog, which includes the commit buttons, then I think it would be better to to have all the buttons also in the tab order, because people are used to being able to tab to the commit buttons. See:
https://learn.microsoft.com/en-us/windows/apps/design/input/keyboard-interactions#use-multiple-tab-stops-and-arrow-keys-for-buttons

(The winui 3 gallery app contains an example of the content dialog, which has this keyboard navigation.)

**Comment by DilsonsPickles** [comment]
I agree. This is something that we have to work on with the Musescore team as the modal component is shared between the two applications. 

We will follow the Musescore convention and then change it for both.
